runtime/acala/src/lib.rs

```diff
@@ -921,6 +927,8 @@ impl orml_vesting::Config for Runtime {
 parameter_types! {
 	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
 	pub const MaxScheduledPerBlock: u32 = 10;
+	// Retry a scheduled item every 10 blocks (1 minute) until the preimage exists.
+	pub const NoPreimagePostponement: Option<u32> = Some(10);
```

Comment:
10 blocks is 2 mins for us. Use `MINUTES` to ensure it is 1 minute.
But I think we can make it 5 minutes here.

Reply:
change to 25 blocks, 25 * 12 = 300s = 5 min